apps/frontend/src/features/leaderboard/components/InfoLeaderBoardModal.tsx

```
@@ -0,0 +1,17 @@
+import { css } from '@emotion/react';
+
+import { MarkdownRenderer } from '@/comp/MarkdownRenderer';
+
+import { leaderboardInfoContent } from './leaderboardInfo';
+
+export const InfoLeaderBoardModal = () => (
+  <div css={containerStyle} tabIndex={0} role="region" aria-label="리더보드 정보">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)